src/components/views/voip/IncomingCallBox.js

```diff
@@ -59,6 +62,7 @@ module.exports = React.createClass({
             }
         }
 
+        const AccessibleButton = sdk.getComponent('elements.AccessibleButton');
```

Comment:
sdk has not been imported

Reply:
oops, good catch. Now actually tested too...